Lib/json/tool.py

```diff
@@ -26,19 +26,26 @@ def main():
                         help='write the output of infile to outfile')
     parser.add_argument('--sort-keys', action='store_true', default=False,
                         help='sort the output of dictionaries alphabetically by key')
+    parser.add_argument('--jsonlines', action='store_true', default=False,
+                        help='parse input using the jsonlines format')
```

Comment:
Maybe `'parse every input line as separate JSON object'`?

Reply:
Write `whatsnew` in 3.8?